.changeset/funny-dots-yawn.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/react': patch
```

Comment:
I don't think we need a changeset for it. We don't include changeset for docs and tests or changes that only impacts dev env. Do you have any concern with that?

If not, we will add `skip changeset` label and checks should be all green after that yeah let me know if you have any concern 🙌🏻

Reply:
Makes sense, wasn't familiar with the conventions y'all use – removed the changeset file 🙂